### DISCUSSION TITLE
kanbam-style interface

### DISCUSSION THREAD

**Discussion by glauberssoutodossantos** [chatwoot/chatwoot#4928] [Feature Requests]
I would like a kanbam-style interface to track conversation status, like the example below.
It would be interesting for the manager to monitor the amount of chat in each status.

![image](https://user-images.githubusercontent.com/99494469/175551581-91a9b551-cc6f-4344-92bd-1ad65d86afc1.png)

**Comment by HermesMacedo** [comment]
What is the technical possibility? Include this in the Code. $$$

[a comment by an automated account is not shown]
